R/plot_breakpoints.R

```diff
@@ -138,7 +138,7 @@ plot_breakpoints <- function(domainDataFrame = NULL,
   # add domain level information
   if (!all(is.na(geneExons$DESC))) {
     p1 <- p1 + geom_linerange(ymin = geneExons$domain_start, ymax = geneExons$domain_end, size = 6, aes(col = .data$DESC, x = gene)) + labs(color = "Domain") + theme(legend.position = "bottom") +
```

Comment:
```suggestion
    p1 <- p1 + geom_linerange(ymin = geneExons$domain_start, ymax = geneExons$domain_end, size = 6, aes(col = .data$DESC, x = gene)) + 
    labs(color = "Domain") + 
    theme(legend.position = "bottom") +
```
just breaking this line up a bit